executable.go

```diff
@@ -1,15 +1,68 @@
 package selfupdate
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
+	"sync"
 	"time"
 
 	"github.com/fynelabs/selfupdate/internal/osext"
 )
 
+var (
+	exePath           string
+	defaultOldExePath string
+	newExePath        string
+	exeErr            error
+	once              sync.Once
+)
+
+func GetExecutableRealPath() (string, error) {
```

Comment:
In go, we tend to not use Get in the name of accessor. Would you mind dropping it from all the public api?

Reply:
Sure, done in this commit https://github.com/fynelabs/selfupdate/pull/34/commits/6f39931d76861d391f1c4731deb069d668f50eed